gemfiles/Gemfile.td-agent-3.3.0

```diff
@@ -22,10 +22,10 @@ gemspec path: ".."
 gem "fluentd", "1.3.3"
 # https://github.com/treasure-data/omnibus-td-agent/blob/v3.3.0/plugin_gems.rb#L16-L23
```

Comment:
Are these gem versions for testing updated as included versions in published td-agent v3.3.3? If not, what is the source of these versions?